src/main/java/teammates/ui/controller/XlsFileDownloadResult.java

```diff
@@ -19,10 +19,10 @@
 import teammates.common.util.Const;
 import teammates.common.util.StatusMessage;
 
-public class XLSFileDownloadResult extends FileDownloadResult {
+public class XlsFileDownloadResult extends FileDownloadResult {
     private XSSFWorkbook workBook;
 
-    public XLSFileDownloadResult(String destination, AccountAttributes account, List<StatusMessage> status,
+    public XlsFileDownloadResult(String destination, AccountAttributes account, List<StatusMessage> status,
```

Comment:
There are other places in this file where a similar fix should be applied. e.g. XSSFWorkbook